### PR TITLE
feat: add dbt 1.8 support

### PR DESCRIPTION
**Pull request by gaurpulkit**

### CODE REVIEW

File: src/datapilot/core/platforms/dbt/wrappers/manifest/v12/schemas.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes4
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes5
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes6
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Nodes7
+from dbt_artifacts_parser.parsers.manifest.manifest_v12 import Sources
+
+from datapilot.core.platforms.dbt.constants import GENERIC
+from datapilot.core.platforms.dbt.constants import SINGULAR
+
+ManifestNode = Union[
```
Comment (gaurpulkit) [inline src/datapilot/core/platforms/dbt/wrappers/manifest/v12/schemas.py:20]: RPCNode which was present in v11 isn't available here

File: src/datapilot/core/platforms/dbt/wrappers/manifest/v12/wrapper.py [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+                }
+                if seed.columns
+                else None
+            ),
+            meta=seed.meta,
+            group=seed.group,
+            docs=seed.docs.dict() if seed.docs else None,
+            patch_path=seed.patch_path,
+            build_path=seed.build_path,
+            deferred=False,  # seed.deferred isn't present in ManifestV12
```
Comment (gaurpulkit) [inline src/datapilot/core/platforms/dbt/wrappers/manifest/v12/wrapper.py:328]: @suryaiyer95 deferred isn't available in v12

File: src/datapilot/core/platforms/dbt/wrappers/manifest/v12/schemas.py (hunk 1 above)
Comment (suryaiyer95) [reply src/datapilot/core/platforms/dbt/wrappers/manifest/v12/schemas.py:20]: what is this Nodes1?

**Comment by suryaiyer95** [comment]
tests seem to be failing :

E   UserWarning: Field "model_unique_id" has conflict with protected namespace "model_".

File: src/datapilot/core/insights/report.py [hunk 3]
```diff
@@ -8,7 +8,7 @@ def generate_ci_cd_report(insights_data):
     for insight in insights_data:
         print(divider)
         print(f"Project: {insight.package_name}")
-        print(f"Model ID: {insight.model_unique_id}")
+        print(f"Model ID: {insight.altimate_unique_id}")
```
Comment (suryaiyer95) [inline src/datapilot/core/insights/report.py:11]: Let's revert this?

**Pull request closed without merging**